### PR TITLE
Added documentation and logger into model

### PR DESCRIPTION
**Pull request by pipo411**

### CODE REVIEW

**Comment by FranzVidangosPari** [comment]
The names of your variables are differents to the documentation

**Review by FranzVidangosPari — COMMENTED** [review]
The names of your variables are differents to the documentation

File: src/com/jalasoft/shopping_car/model/item.py [hunk 1]
```diff
@@ -1,31 +1,87 @@
...
+
 class Item:
+    """
+    Create item objects.
+    """
 
     def __init__(self, name="", price=0, quantity=0):
+        """
+        Item constructor.
+        :param name:        str     item_name
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:15]: The names of the variables do not match the documentation. for item , name and quantity

File: src/com/jalasoft/shopping_car/model/item.py [hunk 2]
```diff
@@ -1,31 +1,87 @@
...
+        LOG.info('Enter on Item constructor')
         self.__id = 0
         self.__name = name
         self.__price = price
         self.__quantity = quantity
 
     def get_name(self):
+        """
+        Function return item_name
+        :return:    str item_name
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:28]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 3]
```diff
@@ -1,31 +1,87 @@
...
+        """
+        LOG.info('GET item_name')
         return self.__name
 
     def get_price(self):
+        """
+        Function return item_price
+        :return: float  item_price
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:36]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 4]
```diff
@@ -1,31 +1,87 @@
...
+        """
+        LOG.info('GET item_price')
         return self.__price
 
     def get_quantity(self):
+        """
+        Function return item_quantity
+        :return:    int item_quantity
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:44]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 5]
```diff
@@ -1,31 +1,87 @@
...
+        """
+        LOG.info('GET item_quantity')
         return self.__quantity
 
     def get_id(self):
+        """
+        Function return item_id
+        :return:    str item_id
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:52]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 6]
```diff
@@ -1,31 +1,87 @@
...
+        """
+        Function set the item_id
+        :param item_id:      str item_id
+        """
+        LOG.info('SET item_id')
+        self.__id = item_id
 
     def set_name(self, name):
+        """
+        Function set the item_name
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:67]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 7]
```diff
@@ -1,31 +1,87 @@
...
+        :param id:      str item_name
+        """
+        LOG.info('SET item_name')
         self.__name = name
 
     def set_price(self, price):
+        """
+        Function set the item_price
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:75]: The names of the variables do not match the documentation.

File: src/com/jalasoft/shopping_car/model/item.py [hunk 8]
```diff
@@ -1,31 +1,87 @@
...
+        :param id: float item_price
+        """
+        LOG.info('SET item_price')
         self.__price = price
 
     def set_quantity(self, quantity):
+        """
+        Function set the item_quantity
```
Comment (FranzVidangosPari) [inline src/com/jalasoft/shopping_car/model/item.py:83]: The names of the variables do not match the documentation.

**Review by FranzVidangosPari — CHANGES_REQUESTED** [review]

File: src/com/jalasoft/shopping_car/model/item.py (hunk 2 above)
Comment (pipo411) [reply src/com/jalasoft/shopping_car/model/item.py:28]: It is only a description for item name

File: src/com/jalasoft/shopping_car/model/item.py (hunk 3 above)
Comment (pipo411) [reply src/com/jalasoft/shopping_car/model/item.py:36]: It is only a description for item name

**Pull request merged**